src/c_oper1.c

```diff
@@ -3726,6 +3791,7 @@ static Obj  HdlrFunc1 (
       fi;
       narg := LEN_LIST( flags );
       methods := METHODS_OPERATION( opr, narg );
+      ;
```

Comment:
These `;` lines come from `T_EMPTY` statements, which in turn are leftovers of `if IsHPCGAP then ... fi;` statements which got optimized away.

We could suppress this in various ways: For starters, we could change `T_EMPTY` statements to not print anything.
Cleaner would of course not to emit them in the first place, but doing this right is a bit tricky, and way beyond the scope of this PR.